src/components/Table/components/SortingIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Colors } from '../../../essentials';
+import { IconProps } from '../../../icons';
+
+type Props = IconProps & {
+    direction: 'ASC' | 'DESC' | '';
```

Comment:
Yeah, makes sense.

Reply:
@div-Leo this won't be exposed externally, this prop is only used internally so there's no conflict with existing API's or any other mental model.

It's true that `''` would make it easier for truthyness check, but we don't need it, since we control it the `active` boolean already.

Reply:
Done!